File: implementation/scipamato/scipamato-core-web/src/test/java/ch/difty/scipamato/core/web/resources/jasper/PaperSummaryReportResourceReferenceTest.java
package ch.difty.scipamato.core.web.resources.jasper;

public class PaperSummaryReportResourceReferenceTest
    extends JasperReportResourceReferenceTest<PaperSummaryReportResourceReference> {

    @Override
    protected PaperSummaryReportResourceReference getResourceReference() {
        return PaperSummaryReportResourceReference.get();
    }

    @Override
    protected String getReportBaseName() {
        return "paper_summary_A4";
    }

    @Override
    protected String getResourceReferencePath() {
        return "ch.difty.scipamato.core.web.resources.jasper.PaperSummaryReportResourceReference";
    }
}

File: core/core-web/src/test/kotlin/ch/difty/scipamato/core/web/resources/jasper/PaperSummaryReportResourceReferenceTest.kt
package ch.difty.scipamato.core.web.resources.jasper

internal class PaperSummaryReportResourceReferenceTest :
    JasperReportResourceReferenceTest<PaperSummaryReportResourceReference>() {

    override val resourceReference: PaperSummaryReportResourceReference
        get() = PaperSummaryReportResourceReference.get()

    override val reportBaseName: String
        get() = "paper_summary_A4"

    override val resourceReferencePath: String
        get() = "ch.difty.scipamato.core.web.resources.jasper.PaperSummaryReportResourceReference"
}
